<ui version="4.0" >
 <class>passwordDialogClass</class>
 <widget class="QDialog" name="passwordDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Enter your password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="passwordLabel" >
     <property name="text" >
      <string>Your password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordEdit" >
     <property name="maxLength" >
      <number>16</number>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveBox" >
     <property name="text" >
      <string>Save password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>OK</string>
     </property>
     <property name="icon" >
      <iconset resource="../../qutim.qrc" >
       <normaloff>:/icons/crystal_project/apply.png</normaloff>:/icons/crystal_project/apply.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../qutim.qrc" />
  <include location="../../../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>passwordDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>273</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
